### ISSUE TITLE
Products: App freezes (ANR) after opening virtual product

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-android#2052]
**Describe the bug**

The app consistently stops responding when I open virtual products in the Products tab. Other products work fine, but as soon as I open a virtual product I can't scroll or exit that screen.

Error in `com.woocommerce.android/.ui.main.MainActivity`:
```
Input dispatching timed out (Waiting to send non-key event because the touched window has not finished processing certain input events that were delivered to it over 500.0ms ago.  Wait queue length: 2.  Wait queue head age: 6166.2ms.)
```

**To Reproduce**
1. Go to the Products tab.
2. Select a virtual product.
3. Try to scroll, edit, or exit the product details.

**Mobile Environment**
Please include:
- Device: moto e5 play
- Android version: 8.1.0
- WooCommerce Android version: 3.8-rc-1

**Comment by nbradbury** [comment]
I'm able to repro this consistently. More details from logcat:

```
2020-03-13 13:33:36.203 1432-2136/? E/ActivityManager: ANR in com.woocommerce.android (com.woocommerce.android/.ui.main.MainActivity)
    PID: 21920
    Reason: Input dispatching timed out (Waiting because the touched window's input channel is full.  Outbound queue length: 1.  Wait queue length: 52.)
    Parent: com.woocommerce.android/.ui.main.MainActivity
```

**Comment by nbradbury** [comment]
Note to self: looks like the product model is being continually modified by adding "Virtual" to its name. With other products we simply show the product name, but virtual products add the word "Virtual" to the display name.

<img width="567" alt="Screen Shot 2020-03-13 at 1 58 39 PM" src="https://user-images.githubusercontent.com/3903757/76647284-e7cd8f00-6532-11ea-9bea-09bd1645d166.png">

**Comment by nbradbury** [comment]
Closed via #2061
